packages/jira-adapter/src/filters/assets/change_object_type_parents.ts

```diff
@@ -0,0 +1,38 @@
+/*
+*                      Copyright 2023 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import { CORE_ANNOTATIONS, isInstanceElement } from '@salto-io/adapter-api'
+import { FilterCreator } from '../../filter'
+
+
+const filter: FilterCreator = ({ config }) => ({
+  name: 'assetsObjectTypeParentFilter',
+  onFetch: async elements => {
+    if (!config.fetch.enableJSM || !config.fetch.enableJsmExperimental) {
+      return
+    }
+    elements
+      .filter(isInstanceElement)
+      .filter(instance => instance.elemID.typeName === 'AssetsObjectType')
+      .forEach(instance => {
+        if (instance.value.parentObjectTypeId === undefined) {
```

Comment:
maybe add a comment here ? (I'm sure you can think of something better than what I wrote 😅 
```suggestion
        // add reference to schema for root assetsObjectType
        if (instance.value.parentObjectTypeId === undefined) {
```